src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -291,36 +315,89 @@ def _empty_stream(stub):
 
 
 def _status_code_and_message(stub):
-  message = 'test status message'
-  code = 2
-  status = grpc.StatusCode.UNKNOWN # code = 2
+  # Function wide constants
+  _MESSAGE = 'test status message'
```

Comment:
I recognize that it's inconsistent with the way other constants are styled, but behavior-scope constants should be styled (sigh) `just_like_behavior_scope_fields`.